modules/apps/29-fee/ica_test.go

```diff
@@ -0,0 +1,192 @@
+package fee_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	icahosttypes "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/host/types"
+	icatypes "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/types"
+	"github.com/cosmos/ibc-go/v3/modules/apps/29-fee/types"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+)
+
+var (
+	// DefaultOwnerAddress defines a reusable bech32 address for testing purposes
+	DefaultOwnerAddress = "cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs"
+
+	// DefaultPortID defines a resuable port identifier for testing purposes
+	DefaultPortID, _ = icatypes.NewControllerPortID(DefaultOwnerAddress)
+
+	// DefaultICAVersion defines a resuable interchainaccounts version string for testing purposes
+	DefaultICAVersion = string(icatypes.ModuleCdc.MustMarshalJSON(&icatypes.Metadata{
```

Comment:
Would it be useful to create a helper function in types that creates the default ICA version given the connection IDs?

Reply:
Done ✅